### ISSUE TITLE
npm release, roadmap, future, ...

### ISSUE THREAD

**Issue by sebpiq** [audiocogs/aurora.js#31]
Hi!

I needed a JavaScript sound file decoder for [node-web-audio-api](https://github.com/sebpiq/node-web-audio-api), and after a couple of hours trying to figure out how aurora works, I have to say it works great.

I have seen the last npm release is quite old, and as I would prefer to add it as a dependency rather than bundling it, I was wondering if you could publish?

Though the lib works great, I think it deserves some love (no offense), for example the build system seems bit odd to me, and also it seems to be a full framework to build a player ... what about splitting in smaller better documented components?

In short, do you have any roadmap, any plans for the future?

I plan to use the audio-decoding part extensively, so I'd be glad to help taking it out, or repackaging it, writing docs and so on ...
